INFO] [launch]: All log files can be found below /home/will/.ros/log/2023-09-08-15-38-31-734261-will-82590
[INFO] [launch]: Default logging verbosity is set to INFO
Swarm written to /home/will/drl_swarm/install/webots_pkg/share/webots_pkg/worlds/my_world.wbt
robot_description =  <?xml version="1.0" ?>
<robot name="TurtlebotBurger3">
    <webots>
        <plugin type="webots_pkg.my_turtlebot_driver.MyTurtlebotDriver" />
    </webots>
</robot>
tb.name =  tb1
robot_description =  <?xml version="1.0" ?>
<robot name="TurtlebotBurger3">
    <webots>
        <plugin type="webots_pkg.my_turtlebot_driver.MyTurtlebotDriver" />
    </webots>
</robot>
tb.name =  tb2
[INFO] [webots-1]: process started with pid [82592]
[INFO] [robot_state_publisher-2]: process started with pid [82594]
[INFO] [driver-3]: process started with pid [82597]
[INFO] [simple_mapper-4]: process started with pid [82600]
[INFO] [robot_state_publisher-5]: process started with pid [82602]
[INFO] [driver-6]: process started with pid [82605]
[INFO] [simple_mapper-7]: process started with pid [82607]
[INFO] [robot_state_publisher-8]: process started with pid [82609]
[INFO] [driver-9]: process started with pid [82611]
[INFO] [robot_state_publisher-10]: process started with pid [82613]
[INFO] [driver-11]: process started with pid [82616]
[driver-3] Cannot connect to Webots instance, retrying in 0 second...
[driver-6] Cannot connect to Webots instance, retrying in 0 second...
[driver-9] Cannot connect to Webots instance, retrying in 0 second...
[driver-11] Cannot connect to Webots instance, retrying in 0 second...
[robot_state_publisher-2] [INFO] [1694209112.061019017] [cf1.robot_state_publisher]: got segment 
[robot_state_publisher-8] [INFO] [1694209112.063959031] [tb1.robot_state_publisher]: got segment 
[robot_state_publisher-10] [INFO] [1694209112.064423943] [tb2.robot_state_publisher]: got segment 
[robot_state_publisher-5] [INFO] [1694209112.064683669] [cf2.robot_state_publisher]: got segment 
[driver-3] Cannot connect to Webots instance on socket "/tmp/webots-1234/ipc/cf1/extern", retrying in 1 second...
[driver-6] Cannot connect to Webots instance on socket "/tmp/webots-1234/ipc/cf2/extern", retrying in 1 second...
[driver-9] Cannot connect to Webots instance on socket "/tmp/webots-1234/ipc/tb1/extern", retrying in 1 second...
[driver-11] Cannot connect to Webots instance on socket "/tmp/webots-1234/ipc/tb2/extern", retrying in 1 second...
[driver-3] Cannot connect to Webots instance on socket "/tmp/webots-1234/ipc/cf1/extern", retrying in 2 seconds...
[driver-6] Cannot connect to Webots instance on socket "/tmp/webots-1234/ipc/cf2/extern", retrying in 2 seconds...
[driver-9] Cannot connect to Webots instance on socket "/tmp/webots-1234/ipc/tb1/extern", retrying in 2 seconds...
[driver-11] Cannot connect to Webots instance on socket "/tmp/webots-1234/ipc/tb2/extern", retrying in 2 seconds...
^C[WARNING] [launch]: user interrupted with ctrl-c (SIGINT)
[robot_state_publisher-2] [INFO] [1694209141.549251895] [rclcpp]: signal_handler(signal_value=2)
[robot_state_publisher-10] [INFO] [1694209141.549289961] [rclcpp]: signal_handler(signal_value=2)
[robot_state_publisher-5] [INFO] [1694209141.549308162] [rclcpp]: signal_handler(signal_value=2)
[driver-11] [INFO] [1694209141.549311697] [rclcpp]: signal_handler(signal_value=2)
[driver-3] [INFO] [1694209141.549313606] [rclcpp]: signal_handler(signal_value=2)
[driver-9] [INFO] [1694209141.549313903] [rclcpp]: signal_handler(signal_value=2)
[robot_state_publisher-8] [INFO] [1694209141.549363179] [rclcpp]: signal_handler(signal_value=2)
[driver-6] [INFO] [1694209141.549357237] [rclcpp]: signal_handler(signal_value=2)
[simple_mapper-7] Traceback (most recent call last):
[simple_mapper-7]   File "/home/will/drl_swarm/install/webots_pkg/lib/webots_pkg/simple_mapper", line 33, in <module>
[simple_mapper-7]     sys.exit(load_entry_point('webots-pkg', 'console_scripts', 'simple_mapper')())
[simple_mapper-7]   File "/home/will/drl_swarm/build/webots_pkg/webots_pkg/simple_mapper.py", line 175, in main
[simple_mapper-7]     rclpy.spin(simple_mapper)
[simple_mapper-7]   File "/opt/ros/galactic/lib/python3.8/site-packages/rclpy/__init__.py", line 196, in spin
[simple_mapper-7]     executor.spin_once()
[simple_mapper-7]   File "/opt/ros/galactic/lib/python3.8/site-packages/rclpy/executors.py", line 705, in spin_once
[simple_mapper-7]     handler, entity, node = self.wait_for_ready_callbacks(timeout_sec=timeout_sec)
[simple_mapper-7]   File "/opt/ros/galactic/lib/python3.8/site-packages/rclpy/executors.py", line 691, in wait_for_ready_callbacks
[simple_mapper-7]     return next(self._cb_iter)
[simple_mapper-7]   File "/opt/ros/galactic/lib/python3.8/site-packages/rclpy/executors.py", line 588, in _wait_for_ready_callbacks
[simple_mapper-7]     wait_set.wait(timeout_nsec)
[simple_mapper-7] KeyboardInterrupt
[simple_mapper-4] Traceback (most recent call last):
[simple_mapper-4]   File "/home/will/drl_swarm/install/webots_pkg/lib/webots_pkg/simple_mapper", line 33, in <module>
[simple_mapper-4]     sys.exit(load_entry_point('webots-pkg', 'console_scripts', 'simple_mapper')())
[simple_mapper-4]   File "/home/will/drl_swarm/build/webots_pkg/webots_pkg/simple_mapper.py", line 175, in main
[simple_mapper-4]     rclpy.spin(simple_mapper)
[simple_mapper-4]   File "/opt/ros/galactic/lib/python3.8/site-packages/rclpy/__init__.py", line 196, in spin
[simple_mapper-4]     executor.spin_once()
[simple_mapper-4]   File "/opt/ros/galactic/lib/python3.8/site-packages/rclpy/executors.py", line 705, in spin_once
[simple_mapper-4]     handler, entity, node = self.wait_for_ready_callbacks(timeout_sec=timeout_sec)
[simple_mapper-4]   File "/opt/ros/galactic/lib/python3.8/site-packages/rclpy/executors.py", line 691, in wait_for_ready_callbacks
[simple_mapper-4]     return next(self._cb_iter)
[simple_mapper-4]   File "/opt/ros/galactic/lib/python3.8/site-packages/rclpy/executors.py", line 588, in _wait_for_ready_callbacks
[simple_mapper-4]     wait_set.wait(timeout_nsec)
[simple_mapper-4] KeyboardInterrupt
[INFO] [driver-3]: process has finished cleanly [pid 82597]
[ERROR] [driver-9]: process has died [pid 82611, exit code 1, cmd '/opt/ros/galactic/lib/webots_ros2_driver/driver --ros-args --params-file /tmp/launch_params_9m99aa4f'].
[ERROR] [driver-6]: process has died [pid 82605, exit code 1, cmd '/opt/ros/galactic/lib/webots_ros2_driver/driver --ros-args --params-file /tmp/launch_params_3hq71n9w'].
[ERROR] [driver-11]: process has died [pid 82616, exit code 1, cmd '/opt/ros/galactic/lib/webots_ros2_driver/driver --ros-args --params-file /tmp/launch_params_c46zcnmv'].
[INFO] [webots-1]: process has finished cleanly [pid 82592]
[INFO] [robot_state_publisher-5]: process has finished cleanly [pid 82602]
[INFO] [robot_state_publisher-10]: process has finished cleanly [pid 82613]
[INFO] [robot_state_publisher-8]: process has finished cleanly [pid 82609]
[INFO] [robot_state_publisher-2]: process has finished cleanly [pid 82594]
[simple_mapper-7] Exception ignored in: <function Executor.__del__ at 0x7fe3cf55c310>
[simple_mapper-7] Traceback (most recent call last):
[simple_mapper-7]   File "/opt/ros/galactic/lib/python3.8/site-packages/rclpy/executors.py", line 229, in __del__
[simple_mapper-7]   File "/opt/ros/galactic/lib/python3.8/site-packages/rclpy/signals.py", line 41, in destroy
[simple_mapper-7] AttributeError: 'NoneType' object has no attribute 'rclpy_unregister_sigint_guard_condition'
[simple_mapper-7] Exception ignored in: <function SignalHandlerGuardCondition.__del__ at 0x7fe3cf5454c0>
[simple_mapper-7] Traceback (most recent call last):
[simple_mapper-7]   File "/opt/ros/galactic/lib/python3.8/site-packages/rclpy/signals.py", line 31, in __del__
[simple_mapper-7] TypeError: catching classes that do not inherit from BaseException is not allowed
[simple_mapper-4] Exception ignored in: <function Executor.__del__ at 0x7f57756a7310>
[simple_mapper-4] Traceback (most recent call last):
[simple_mapper-4]   File "/opt/ros/galactic/lib/python3.8/site-packages/rclpy/executors.py", line 229, in __del__
[simple_mapper-4]   File "/opt/ros/galactic/lib/python3.8/site-packages/rclpy/signals.py", line 41, in destroy
[simple_mapper-4] AttributeError: 'NoneType' object has no attribute 'rclpy_unregister_sigint_guard_condition'
[simple_mapper-4] Exception ignored in: <function SignalHandlerGuardCondition.__del__ at 0x7f57756884c0>
[simple_mapper-4] Traceback (most recent call last):
[simple_mapper-4]   File "/opt/ros/galactic/lib/python3.8/site-packages/rclpy/signals.py", line 31, in __del__
[simple_mapper-4] TypeError: catching classes that do not inherit from BaseException is not allowed
[ERROR] [simple_mapper-4]: process has died [pid 82600, exit code -2, cmd '/home/will/drl_swarm/install/webots_pkg/lib/webots_pkg/simple_mapper --ros-args -r __ns:=/cf1'].
[ERROR] [simple_mapper-7]: process has died [pid 82607, exit code -2, cmd '/home/will/drl_swarm/install/webots_pkg/lib/webots_pkg/simple_mapper --ros-args -r __ns:=/cf2'].
